editor.planx.uk/src/ui/editor/SettingsSection.tsx

```diff
@@ -21,26 +14,16 @@ const Root = styled(Box, {
   "& > * + *, & > form > * + *": {
     ...contentFlowSpacing(theme),
   },
-  ...(background && {
+  ...(bgcolor && {
     background: theme.palette.background.paper,
     marginTop: theme.spacing(2),
     padding: theme.spacing(2.5),
     border: `1px solid ${theme.palette.border.light}`,
   }),
 }));
 
-export default function SettingsSection({
-  children,
-  background,
-  testId,
-}: {
-  children: ReactNode;
-  background?: boolean;
-  testId?: string;
-}) {
-  return (
-    <Root data-testid={testId} background={background}>
-      {children}
-    </Root>
-  );
+export default function SettingsSection(
+  props: { children: ReactNode } & Partial<BoxProps>,
```

Comment:
@DafyddLlyr change that we discussed on our call but didn't resolve, I've now got it all working and it is showing up in the DOM as expected.